fluent-jdbc/src/main/java/org/codejargon/fluentjdbc/internal/mappers/DefaultObjectMapperRsExtractors.java

```diff
@@ -90,7 +99,11 @@ private static void binaryTypes(Map<Class, ObjectMapperRsExtractor<?>> exs) {
                 return null;
             }
             ByteBuffer data = ByteBuffer.wrap(blob.getBytes(0, (int) blob.length()));
-            blob.free();
+            try{
+              blob.free();
```

Comment:
is it necessary to call this free() ? What are the consequences if we don't?

Changed catch Exception to catch SQLFeatureNotSupportedException for now. If we must keep the .free(), we need to make sure this attempt to call free() happens only once. Otherwise it's a performance issue (an exception each time the extractor works.